### PR TITLE
Revert "Improve notices endpoint"

### PR DESCRIPTION
**Pull request by samhotep**
- Reverts canonical/ubuntu-com-security-api#146

- Adds indexes

### CODE REVIEW

File: migrations/versions/c27fec1bfee2.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""empty message
```
Comment (jpmartinspt) [inline migrations/versions/c27fec1bfee2.py:1]: There is already a migration for this in the repo, right?

Comment (jpmartinspt) [reply migrations/versions/c27fec1bfee2.py:1]: https://github.com/canonical/ubuntu-com-security-api/blob/main/migrations/versions/c27fec1bfee2_.py

File: webapp/views.py [hunk 2]
```diff
@@ -412,6 +412,7 @@ def get_notices(**kwargs):
     release = kwargs.get("release")
     limit = kwargs.get("limit", 20)
```
Comment (jpmartinspt) [inline webapp/views.py:413]: I would drop this one to `10` just in case.

File: migrations/versions/c27fec1bfee2.py (hunk 1 above)
Comment (samhotep) [reply migrations/versions/c27fec1bfee2.py:1]: Yes, it's the same one actually

**Comment by samhotep** [comment]
Closing as revert was applied [here](https://github.com/canonical/ubuntu-com-security-api/pull/159)

**Pull request closed without merging**